.github/actions/build-debezium-ibmi/action.yml

```
@@ -0,0 +1,47 @@
+name: "Build IBMi"
+description: "Builds the Debezium IBMi connector"
+
+inputs:
+  path-core:
+    description: "Debezium core repository checkout path"
+    required: true
+  path-ibmi:
+    description: "Debezium IBMi repository checkout path"
+    required: true
+  shell:
+    description: "The shell to use"
+    required: false
+    default: bash
+
+runs:
+  using: "composite"
+  steps:
+    - name: Build Debezium (Core)
+      shell: ${{ inputs.shell }}
+      run: >
+        ./${{ inputs.path-core }}/mvnw clean install -B -ntp -f ${{ inputs.path-core }}/pom.xml
+        -pl debezium-assembly-descriptors,debezium-bom,debezium-core,debezium-embedded,:debezium-ide-configs,:debezium-checkstyle,:debezium-revapi
+        -am
+        -DskipTests=true
+        -DskipITs=true
+        -Dcheckstyle.skip=true
+        -Dformat.skip=true
+        -Drevapi.skip
+        -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
+        -Dmaven.wagon.http.pool=false
+        -Dmaven.wagon.httpconnectionManager.ttlSeconds=120
+
+    - name: Build IBMi
+      shell: ${{ inputs.shell }}
+      run: >
+        ./${{ inputs.path-core }}/mvnw clean install -B -ntp -f ${{ inputs.path-ibmi }}/pom.xml 
+        -Passembly
+        -Dcheckstyle.skip=true
+        -Dformat.skip=true
+        -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
+        -Dmaven.wagon.http.pool=false
+        -Dmaven.wagon.httpconnectionManager.ttlSeconds=120
+        -Ddebezium.test.records.waittime=5
+        -Ddebezium.test.records.waittime.after.nulls=5        
+        -DfailFlakyTests=false
+        -DskipITs
```

```suggestion
        -DskipITs
        -Ddebezium.test.mongo.replica.primary.startup.timeout.seconds=120
```
Just so that one pesky MongoDB unit test in the testcontainers module doesn't cause havoc :)